specifications/1614618-timelines.md

```diff
@@ -0,0 +1,95 @@
+- Feature Name: Timelines
```

Comment:
Note that we have another feature proposal with the same name:
https://bugzilla.mozilla.org/show_bug.cgi?id=1473760

I suggest we rename this feature to "Milestones".